src/chronify/time_series_mapper_representative.py

```diff
@@ -45,7 +47,12 @@ def _check_source_table_has_time_zone(self) -> None:
             msg = f"time_zone is required for tz-aware representative time mapping and it is missing from source table: {self._from_schema.name}"
```

Comment:
@daniel-thom - I think we need to check time_zone a different way here since list_columns() no longer captures all table columns.

Reply:
We do have an assert for time_zone in _apply_mapping() so we can consolidate it.